bindings/wasm/examples/browser/private_tangle.js

```diff
@@ -0,0 +1,64 @@
+// Copyright 2020-2021 IOTA Stiftung
+// SPDX-License-Identifier: Apache-2.0
+
+import { Client, Config, Document, KeyType, Network } from "../../web/identity_wasm.js";
+import {
+    logObjectToScreen,
+    logToScreen,
+} from "./utils.js";
+
+/**
+    This example shows how a DID document can be created on a private tangle.
+    It can be run together with a local hornet node.
+    Refer to https://github.com/iotaledger/one-click-tangle/tree/chrysalis/hornet-private-net
+    for setup instructions.
+**/
+export async function createIdentityPrivateTangle(inBrowser = true, log = true) {
+    if (log) logToScreen("Identity creation started...");
+    if (log) logToScreen("This might take a few seconds to complete proof of work!");
+
+    let restURL
+    let networkName
+
+    if (inBrowser) {
+        // Get the required parameters from the input fields
+        restURL = document.querySelector("#create-private-rest-url").value;
+        networkName = document.querySelector("#create-private-network-name").value;
+    } else {
+        restURL = "http://127.0.0.1:14265/";
+        networkName = "priv";
```

Comment:
Standardise example network names to `"custom"`.

```suggestion
        networkName = "custom";
```